frontend/src/theme/utils.ts

```diff
@@ -239,18 +240,25 @@ export const createEmotionColors = (genericColors: {
   tableGray: genericColors.gray40,
 })
 
-const createEmotionTheme = (
+export const createEmotionTheme = (
   themeInput: Partial<ICustomThemeConfig>,
   baseThemeConfig = baseTheme
 ): Theme => {
+  const { genericColors, genericFonts, fonts } = baseThemeConfig.emotion
+
   const { name, font, ...customColors } = themeInput
+  const parsedFont = font
```

Comment:
I think there's a fun/subtle bug here that ends up not manifesting anywhere for now because some later behavior with baseweb defaults masks it.

Since the proto enum value for `SANS_SERIF` is 0, if the theme config explicitly picks the `sansSerif` font, this falsy check should set `parsedFont = undefined`, but since undefined fonts get set to the baseWeb default, the value for `bodyFont`/`headingFont` ends up being set to `sansSerif` anyway, which is what we want in the end.

We probably want to check `font !== undefined` here and add a comment mentioning that `CustomThemeConfig.FontFamily.SANS_SERIF === 0` so fails falsy checks.

Reply:
oh good point!  will update that!